CppSamples/ChilliConnectCppDemo/HttpSystem.cpp

```diff
@@ -0,0 +1,153 @@
+#include "HttpSystem.h"
+
+#include <iostream>
+#include <string>
+#include <Windows.h>
+#include <winhttp.h>
+#include <codecvt>
+#include <locale>
+
+HttpSystem::HttpSystem()
+{
+}
+
+HttpSystem::~HttpSystem()
+{
+}
+
+wstring
+HttpSystem::CreateHeaderString(const map<string,string>& headers)
+{
+	string headerBlob = "Content-Type: application/json\r\n";
+
+	for (auto it = headers.begin(); it != headers.end(); ++it)
```

Comment:
I thought you might have recognised that from ChilliSource :D that was a copy paste job ;)